scss/main.scss

```diff
@@ -1535,4 +1535,171 @@ p.cardText {
 
 .hidden {
   visibility: hidden;
+}
+
+.starting-projects-container {
+  transition: all ease .2s;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: min-content;
+  max-width: 1400px;
+  padding: 80px 96px;
+  margin-right: auto;
+  margin-left: auto;
+  .starting-projects-info {
+    transition: all ease .2s;
+    position: relative;
+    text-align: center;
+    max-width: 600px;
+    height: max-content;
+    h2 {
+      transition: all ease .2s;
+      color: $darker-grey;
+      height: min-content;
+      padding-top: 0px;
+      margin-bottom: 16px;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 3rem;
+    }
+    p {
+      transition: all ease .2s;
+      margin-bottom: 24px;
+      color: $darker-grey;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 1.25rem;
+    }
+  }
+  .starting-projects-grid {
+    transition: all ease .2s;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 32px;
+    margin-top: 24px;
+    width: 100%;
+    height: min-content;
+  }
+  .starting-projects-group {
+    transition: all ease .2s;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    height: min-content;
+    width: 31.45%;
+    .starting-projects-info {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      height: min-content;
+      max-height: 100%;
+      width: 100%;
+      text-align: left;
+    }
+    h2 {
```

Comment:
Looks like there's some duplication with these h2 and p styles.
You are styling .starting-projects-info > h2 and p earlier in the file.